### DISCUSSION TITLE
Autocomplete stoped working with table alias

### DISCUSSION THREAD

**Discussion by Rosada-eng** [dbeaver/dbeaver#35384] [Q&A] [answered]
### Description

After upgrading Dbeaver to the latest version (24.1.5.202408181605), columns autocomplete stoped working when using alias. 

![autocomplete-does-not-work](https://github.com/user-attachments/assets/1a4489bf-c86a-4971-b546-0e6d90ff052d)

It works perfectly fine when I don't use alias.

![autocomplete-works](https://github.com/user-attachments/assets/a7dbc9aa-9c9d-44cc-be62-7f5af22125ab)

I have already enabled to "Use Global Searching" on database connection.

![dbeaver-config](https://github.com/user-attachments/assets/2988cd82-a8fe-45d7-ae5d-537f6cb5fde9)

I even tried to downgrade dbeaver version, but the problem still goes on. 

Any suggestions?

### DBeaver Version

Community Edition 24.1.5.202408181605 (then, downgraded to 24.0.5)

### Operating System

macOS Sonoma 14.5

### Database and driver

PostgreSQL 16.1

**Comment by E1izabeth** [comment, marked as answer]
Please remove empty lines between parts of query or change ‘Blank line is statement’ elimiter setting from ‘Always’ to ‘Smart’.

**Comment by Rosada-eng** [comment]
That worked fine, Elizabeth! Thanks so much for helping!
